.vscode/settings.json

```diff
@@ -15,5 +15,10 @@
 	],
 	"eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
 	"typescript.tsdk": "node_modules/typescript/lib",
-	"cSpell.words": ["menuitemcheckbox"]
+	"[javascript]": {
+		"editor.defaultFormatter": "prettier.prettier-vscode"
+	},
+	"[markdown]": {
+		"editor.defaultFormatter": "prettier.prettier-vscode"
+	}
```

Comment:
I believe these are unnecessary since `editor.defaultFormatter` is set in the root? I'll remove, but please yell at me if I'm wrong - I can add them back (or approve a PR to) if they're actually necessary.